azure-pipelines.yml

```diff
@@ -0,0 +1,182 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  branches:
+    include:
+      - master
+      - azure-pipelinestest
```

Comment:
I will change this branch trigger in a later PR. I wan't to make sure that there is smooth integration. So I don't want to take off this branch till we can validate everything works. Then I will put in another PR to get rid of this trigger.